analysis/rendering.py

```diff
@@ -20,7 +20,7 @@
 from common import utils
 
 
-def render_report(experiment_results, template, in_progress):
+def render_report(experiment_results, template, git_hash, in_progress):
```

Comment:
I think this should come from the database, so it should be in experiment_results.

Reply:
Moved.